src/components/sideMenu/index.js

```diff
@@ -61,38 +77,103 @@ function SideMenu() {
         variant="permanent"
       >
         <div className={navigation}>
-          <p className={list}>
-            <IconButton className={link} onClick={() => history.push(orderHistory)}>
-              <Icon className={icon}>
-                <History />
-              </Icon>
-              Order History
-            </IconButton>
-          </p>
-          <p className={list}>
-            <IconButton className={link} onClick={() => history.push(vendors)}>
-              <Icon className={icon}>
-                <PeopleAlt />
-              </Icon>
-              Vendors
-            </IconButton>
-          </p>
-          <p className={list}>
-            <IconButton className={link} onClick={() => history.push(users)}>
-              <Icon className={icon}>
-                <PeopleAlt />
-              </Icon>
-              Users
-            </IconButton>
-          </p>
-          <p className={list}>
-            <IconButton className={link} onClick={() => history.push(categories)}>
-              <Icon className={icon}>
-                <Category />
-              </Icon>
-              Categories
-            </IconButton>
-          </p>
+          {role === admin ? (
+            <>
+              {' '}
+              <p className={list}>
+                <IconButton className={link} onClick={() => history.push('/orderhistory')}>
```

Comment:
This all side Nav based on the user should be implemented in seprate files and also used RouteName Constants and Link why are you using onclick handler here ??